### PR TITLE
Docker, kubernetes and helm support

### PR DESCRIPTION
**Pull request by molgergo01**

### CODE REVIEW

File: charts/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      {{- include "bbg-pictures-repository-backend.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "bbg-pictures-repository-backend.labels" . | nindent 8 }}
+	{{- with .Values.podLabels }}
```
Comment (SabianRobi) [inline charts/templates/deployment.yaml]: sus whitespace, replace with spaces like at every other line

File: scripts/helm_install.sh [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+helmInstall() {
+  cd ${PROJECT_ROOT_DIRECTORY}/charts
+
+  helm package . --version 0.0.0
+  helm upgrade --install bbg-pictures-repository-backend ./bbg-pictures-repository-backend-0.0.0.tgz --set image.tag=${VERSION}
+}
+
+buildImage
+helmInstall
```
Comment (SabianRobi) [inline scripts/helm_install.sh]: Put an empty line at the end please

**Review by SabianRobi — COMMENTED** [review]
Check syntax

File: charts/templates/deployment.yaml (hunk 1 above)
Comment (molgergo01) [reply charts/templates/deployment.yaml]: Done

File: scripts/helm_install.sh (hunk 2 above)
Comment (molgergo01) [reply scripts/helm_install.sh]: Done

**Pull request merged**